js/signin.js

```diff
@@ -20,45 +20,52 @@ emailInput.addEventListener("blur", function () {
   emailInput.classList.remove("error-border");
 });
 
-const passwordErrorMessage = document.querySelector(".password-error-message");
+const passwordErrorMessages = document.querySelector(".password-error-message");
 const passwordInput = document.querySelector(".password-input");
 
 passwordInput.addEventListener("blur", function () {
   if (passwordInput.value === "") {
-    passwordErrorMessage.textContent = "비밀번호를 입력해 주세요.";
+    passwordErrorMessages.textContent = "비밀번호를 입력해 주세요.";
     passwordInput.classList.add("error-border");
     return;
   }
 
-  passwordErrorMessage.textContent = "";
+  passwordErrorMessages.textContent = "";
   passwordInput.classList.remove("error-border");
 });
 
-
-const eyeIconChange = document.querySelector(".eye-icon-change");
-
-eyeIconChange.addEventListener("click", function () {
-  if (passwordInput.type === "password"){
-    eyeIconChange.src = "./image/eye-on.png";
-    passwordInput.type = "text";
-    return;
-  }
-  eyeIconChange.src = "./image/eye-off.png";
-    passwordInput.type = "password";
-});
-
 const loginForm = document.querySelector(".login-form");
 
-loginForm.addEventListener("submit", function (event) {
+async function login(event){
   event.preventDefault();
-  if (emailInput.value !== "test@codeit.com" || passwordInput.value !== "codeit101") {
+    
+  const isMaster = emailInput.value === "test@codeit.com" && passwordInput.value === "codeit101";
+  
+  if (isMaster === false) {
     emailErrorMessage.textContent = "이메일을 확인해 주세요.";
-    passwordErrorMessage.textContent = "비밀번호를 확인해 주세요.";
+    passwordErrorMessages.textContent = "비밀번호를 확인해 주세요.";
     emailInput.classList.add("error-border");
     passwordInput.classList.add("error-border");
     return;
   }
-
-  window.location.href = "./folder.html";
   
-});
+  const testLogin = {
+    email : 'test@codeit.com',
+    password : 'sprint101',
+  }
+
+  const response = await fetch ('https://bootcamp-api.codeit.kr/docs/api/sign-in', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(testLogin),
+  });
+
+  if (response.status === 200) {
+    const responseData = await response.json();
+    window.location.href = "./folder.html";
+  }
```

Comment:
status로 응답이 정상적인지 체크할 수도 있지만 보통은 ok라는 값을 이용해서 해주는게 무난한 것 같아요~
ok는 status가 200번대라면 true값으로 표현이 되어요~
```suggestion
  if (response.ok) {
    const responseData = await response.json();
    window.location.href = "./folder.html";
  }
```

Reply:
로그인을 했을 때, 로그인 상태를 유지하기 위해서 responseData라는 변수에는 accessToken, refreshToken과 같은 토큰값이 담기게 되는데요~ 이 값도 로컬스토리지에 저장해보면 좋을 것 같네요!
```suggestion
  if (response.status === 200) {
    const responseData = await response.json();
    localStorage.setItem('accessToken', responseData.data.accessToken);
    
    window.location.href = "./folder.html";
  }
```

Reply:
folder.html 페이지로 이동했을 때, 화면에 정상적으로 보여지려면 folder.html 파일도 만들어주는게 좋을 것 같네요!